LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/IoTHubRegistryManager.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaTools
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Shared;
+
+    public sealed class IoTHubRegistryManager : IDeviceRegistryManager
+    {
+        private readonly RegistryManager instance;
+
+        public static IDeviceRegistryManager From(RegistryManager instance)
+        {
+            return new IoTHubRegistryManager(instance);
+        }
+
+        private IoTHubRegistryManager(RegistryManager instance)
+        {
+            this.instance = instance;
+        }
+
+        public Task<Configuration> AddConfigurationAsync(Configuration configuration)
+        {
+            return this.instance.AddConfigurationAsync(configuration);
+        }
```

Comment:
This and all other similar rows can be simplified to
```csharp
public Task<Configuration> AddConfigurationAsync(Configuration configuration)
   => this.instance.AddConfigurationAsync(configuration);
```

Reply:
Done